### PR TITLE
Chore 1081 update footer

### PR DESCRIPTION
**Pull request by victordelval**
## Description

Closes: regen-network/regen-registry#1081

* 4 cols
* Update some links
* Remove old ones
* Hidden disabled links
* Disabled basket NCT link/check

---

### Author Checklist

_All items are required. Please add a note to the item if the item is not applicable and
please add links to any relevant follow up issues._

I have...

- [ ] provided a link to the relevant issue or specification
- [ ] provided instructions on how to test
- [ ] reviewed "Files changed" and left comments if necessary
- [ ] confirmed all CI checks have passed
- [ ] once the PR is closed, set up backport PRs for `redwood` and `hambach` (see below)

#### Setting up backport PRs

After merging your PR to `master`, set up backports by doing the following:

1. If your branch does not have merge commits, add the following comment to
   your PR, `@Mergifyio backport redwood hambach`.

2. If your branch does have merge commits:

    a. Pull latest `master`, `hambach` and `redwood` branches

    b. Create new branches for backports and merge `master` (replace `<PR#>` with your PR #)
    ```
    git checkout -b hambach-backport-<PR#> hambach
    git merge master
    git push origin hambach-backport-<PR#>
    git checkout -b redwood-backport-<PR#> redwood
    git merge master
    git push origin redwood-backport-<PR#>`
    ```

    c. Open new PRs in regen-web targeting `hambach` and `redwood`, respectively.
  

### How to test

1. Go to registry footer https://deploy-preview-1229--regen-registry.netlify.app/

### Reviewers Checklist

_All items are required. Please add a note if the item is not applicable and please **add
your handle next to the items reviewed if you only reviewed selected items**._

I have...

- [ ] confirmed all author checklist items have been addressed
- [ ] reviewed code correctness and readability
- [ ] verified React components follow DRY principles
- [ ] reviewed documentation is accurate
- [ ] reviewed tests
- [ ] manually tested (if applicable)

### CODE REVIEW

**Comment by victordelval** [comment]
@erikalogie , you can check here. Thanks!

File: web-registry/src/components/organisms/AppFooter.tsx [hunk 1]
```diff
@@ -6,71 +6,95 @@ import {
...
+          href: '/projects',
         },
+        {
+          title: 'Credit Classes',
+          href: '/',
+        },
+        // {
+        //   title: 'Baskets',
+        //   href: '/',
+        // },
```
Comment (flagrede) [inline web-registry/src/components/organisms/AppFooter.tsx:30]: Should we remove all these commented items?
We can always check history if we need to get them back.

File: web-registry/src/components/organisms/AppFooter.tsx [hunk 2]
```diff
@@ -6,71 +6,95 @@ import {
...
+          href: `/storefront`,
         },
+        // {
+        //   title: 'My Orders',
+        //   href: `/`,
+        // },
+        // {
+        //   title: 'Trade History',
+        //   href: `/`,
+        // },
```
Comment (flagrede) [inline web-registry/src/components/organisms/AppFooter.tsx:47]: same

File: web-registry/src/components/organisms/AppFooter.tsx [hunk 3]
```diff
@@ -6,71 +6,95 @@ import {
...
+          href: '/stats/activity',
         },
+        // {
+        //   title: 'Ownership List',
+        //   href: '/',
+        // },
+        // {
+        //   title: 'Ecocredit Batches',
+        //   href: '/',
+        // },
```
Comment (flagrede) [inline web-registry/src/components/organisms/AppFooter.tsx:64]: same

File: web-registry/src/components/organisms/AppFooter.tsx [hunk 4]
```diff
@@ -6,71 +6,95 @@ import {
...
+  //     footerItems[0].items.unshift({
+  //       title: 'NCT',
+  //       href: '/baskets/eco.uC.NCT',
+  //     });
+  //   }
+  //   footerItems[0].items.push({
+  //     title: 'Activity',
+  //     href: '/stats/activity',
+  //   });
+  // }
```
Comment (flagrede) [inline web-registry/src/components/organisms/AppFooter.tsx:97]: same

File: web-registry/src/components/organisms/RegistryNav.tsx [hunk 5]
```diff
@@ -153,12 +153,12 @@ const RegistryNav: React.FC = () => {
...
-      title: 'NCT',
-      href: '/baskets/eco.uC.NCT',
-    });
-  }
+  // if (nctBasket) {
+  //   menuItems.unshift({
+  //     title: 'NCT',
+  //     href: '/baskets/eco.uC.NCT',
+  //   });
+  // }
```
Comment (flagrede) [inline web-registry/src/components/organisms/RegistryNav.tsx:161]: same

**Review by flagrede — COMMENTED** [review]
Looks good !
Just having one question.

File: web-registry/src/components/organisms/AppFooter.tsx (hunk 1 above)
Comment (victordelval) [reply web-registry/src/components/organisms/AppFooter.tsx]: lgtm!

**Review by mhagel — APPROVED** [review]

**Review by wgwz — APPROVED** [review]

**Comment by erikalogie** [comment]
Hey @victordelval sorry I missed this notification yesterday. 

Desktop looks great, I just noticed some small mobile styling issues: https://monosnap.com/file/yIqjG3lw8kkA3yQ7vcG82IGnX97CuS and https://monosnap.com/file/suf2hb2D3TMITQdX6NQxxw8ATNWsjR.

I know some of these tweaks are partially unrelated to the current task so feel free to open separate issues if necessary.

**Comment by victordelval** [comment]
@Mergifyio backport redwood hambach

**Pull request merged**